README.md

```diff
@@ -1,2 +1,155 @@
-# bedrock-web-data-hub
-Web-based Bedrock Data Hub
+# Web Based Bedrock Data Hub Adapter _(bedrock-web-data-hub)_
+
+[![Build Status](https://travis-ci.org/digitalbazaar/bedrock-web-data-hub.png?branch=master)](https://travis-ci.org/digitalbazaar/bedrock-web-data-hub)
```

Comment:
This package isn't currently being tested on travis-ci. (No config, not setup there yet).

Reply:
Thanks, removed.